frontend/app/controllers/Info.scala

```diff
@@ -66,7 +67,17 @@ trait Info extends Controller {
     )
   }
 
-  def supporterAustralia = CachedAndOutageProtected { implicit request =>
+  def supporterAustralia = NoCacheAction { implicit request =>
```

Comment:
You are disabling the cache for a read-heavy page like the landing page. Are we able to support that traffic? As an alternative, you can implement the test using client-side logic or build a VCL logic around it. https://github.com/guardian/fastly-edge-cache/blob/master/theguardiancom/src/main/resources/varnish21/ab-tests.vcl